### ISSUE TITLE
CI step `check-release-images-existing` fails on non-existing latest tag

### ISSUE THREAD

**Issue by jiaqiluo** [rancher/rancher#41952]
**Probelm** 

The CI step `check-release-images-existing` detects the following missing image tags:

```
ERROR: Summary of missing image(s):
rancher/mirrored-cloud-provider-vsphere-cpi-release-manager:latest
rancher/mirrored-cloud-provider-vsphere-csi-release-driver:latest
rancher/mirrored-cloud-provider-vsphere-csi-release-syncer:latest
rancher/mirrored-sig-storage-csi-attacher:latest
rancher/mirrored-sig-storage-csi-node-driver-registrar:latest
rancher/mirrored-sig-storage-csi-provisioner:latest
rancher/mirrored-sig-storage-csi-resizer:latest
rancher/mirrored-sig-storage-livenessprobe:latest
time="2023-06-12T19:29:56Z" level=fatal msg="exit status 1"
```
ref https://drone-publish.rancher.io/rancher/rancher/9924/5/9

They come from the following chart:
```
rancher-vsphere-cpi:102.1.0+up1.5.1
rancher-vsphere-csi:102.1.0+up3.0.1-rancher1
```

Further investigation shows that the `latest` tags for those images do not exist in the upstream registry, and they are used in the mentioned charts as an indicator that an invalid Kubernetes release version is passed to the chart (See [PR](https://github.com/rancher/vsphere-charts/pull/55)).

**Solution**

To eliminate the error in the CI and the non-existing tags in the generated images.txt in Rancher's release assets, we decided to ignore the `latest` tag on those mentioned charts when generating the images.txt file.

**Comment by jiaqiluo** [comment]
We can validate the fix when the next Rancher RC is cut:

- CI step check-release-images-existing should not report the mentioned errors
- the images.txt file should not contain those mentioned tags

**Comment by jiaqiluo** [comment]
The fix is validated on Rancher  [v2.7.5-rc6](https://github.com/rancher/rancher/releases/tag/v2.7.5-rc6):

Check the images.txt file in the release assets, and do not see the `latest` tags for the images mentioned in the issue.
